dev/com.ibm.ws.classloading.bells_fat/fat/src/com/ibm/ws/classloading/BellFatTest.java

```diff
@@ -69,7 +74,7 @@ public static void tearDown() throws Throwable {
 
     @Before
     public void beforeTest() throws Throwable {
-        server.startServer();
+        // nada
```

Comment:
Why keep this method?  Is it overriding something?

Reply:
I removed the method from all three test cases in the suite.